client/src/components/list/CreateUserList.tsx

```diff
@@ -1,78 +1,104 @@
-import { 
-    Input, 
-    Radio, 
-    Stack,
-    VStack,
-    Button,
-    Select,
-    Heading,
-    FormControl, 
-    FormLabel,
-    FormErrorMessage,
-    FormHelperText,
-    useDisclosure,
-    Center
+import {
+  Input,
+  Radio,
+  Stack,
+  VStack,
+  Button,
+  Select,
+  Heading,
+  FormControl,
+  FormLabel,
+  FormErrorMessage,
+  FormHelperText,
+  useDisclosure,
+  Center,
+  useToast
 } from "@chakra-ui/react";
 
 import * as React from 'react';
-import{ Formik, Form, Field, useFormik } from "formik";
+import { Formik, Form, Field, useFormik } from "formik";
 import * as Yup from 'yup';
 import ChooseRatingModal from "./ChooseRatingModal";
+import { useCreateUserListMutation, useMeQuery } from "../../generated/graphql";
+import { useRouter } from "next/dist/client/router";
 //import { RatingSystem } from "../../generated/graphql";
 
 
 const CreateUserList: React.FC<{}> = () => {
+  const [createUserList] = useCreateUserListMutation();
+  const { data: meData } = useMeQuery();
+  const toast = useToast();
+  const router = useRouter();
 
-    const { isOpen, onOpen, onClose } = useDisclosure()
+  const validationSchema = Yup.object({
```

Comment:
where are the errors displayed if a field is not validated?

Reply:
I see in https://formik.org/docs/guides/validation that you can pass a function accepting `errors` when you use the Formik component, but not seeing how we display errors here